nginx-php-moodle/Dockerfile

```diff
@@ -6,12 +6,11 @@ LABEL version="latest"
 # Disable frontend dialogs
 ENV DEBIAN_FRONTEND noninteractive
 
-ARG MOODLE_DOWNLOAD_FILE
-ARG MOODLE_TRACK
-ARG MOODLE_VERSION
-ARG MOOVE_VERSION
-ARG NGINX_VERSION
-ARG ONELOGIN_PLUGIN_VER
+ARG MOODLE_DOWNLOAD_FILE=moodle-latest-35.tgz
+ARG MOODLE_TRACK=stable35
+ARG MOOVE_VERSION=v0.1.2
+ARG NGINX_VERSION=1.14.0-0+xenial2
+ARG ONELOGIN_PLUGIN_VER=v2.5.1
```

Comment:
Can these values come from the .env file?

Reply:
Not needed in the ENV file because they're only using during build (vs. runtime)

Reply:
Ok that makes sense.